apps/blockchain/config/config.exs

```diff
@@ -31,4 +31,7 @@ use Mix.Config
 
 config :blockchain,
   port: String.to_integer(System.get_env("P2P_PORT") || "5000"),
+  proof_of_work: if(Mix.env == :test, do: Blockchain.Test.ProofOfWork, else: Blockchain.ProofOfWork)
```

Comment:
Don't do this

if you comment out this line 
```elixir
#     import_config "#{Mix.env}.exs"
```
then you need to create the respected file for the environment. Because it will load that file to

What you could do is to put the default configuration on `config/config.exs` and then use `config/test.exs` for override the values.

Side note, the override is using a deep overwriting

Reply:
ok took care of that